examples/text-generation/utils.py

```diff
@@ -203,6 +203,14 @@ def setup_model(args, model_dtype, model_kwargs, logger):
             torch_dtype=model_dtype,
             **model_kwargs,
         )
+    elif args.load_cp:
+        from neural_compressor.torch.quantization import load
```

Comment:
@ulivne sounds like `neural_compressor` is missing from the `requirements` consider adding it!

Reply:
> @ulivne sounds like `neural_compressor` is missing from the `requirements` consider adding it!

neural_compressor is installed automatically as part of habana software stack. it replaces habana_quantization_toolkit which was also not part of requirements.